src/pages/VaultsPage/HasVaults.tsx

```diff
@@ -143,17 +142,16 @@ export default function HasVaults({
   const classes = useStyles();
   const [open, setOpen] = useState<boolean>(false);
   const [openwd, setOpenwd] = useState<boolean>(false);
-  const [openfn, setOpenfn] = useState<boolean>(false);
 
   const handleClick = () => {
     setOpen(!open);
   };
   const handleClickwd = () => {
     setOpenwd(!open);
```

Comment:
while you're in this file, can you take a look at this? that's a bug, it's not using `openwd`

Reply:
although, you know, you could argue that only one of these modals should be open at any given time, so you could use a single `useState` for all of them that is set to either a blank string (all modals closed) or a short string e.g. 'deposit' to open the deposit modal